contracts/Conference.sol

```diff
@@ -252,6 +226,20 @@ contract Conference is GroupAdmin {
         attendanceMaps = _maps;
         ended = true;
         endedAt = now;
+
+        // calculate total attended
+        totalAttended = 0;
```

Comment:
Do we need to set it to `0` here? Isn't the default value of int 0 anyway? If you want to make it explicit, maybe you can set it when it is declared?

Reply:
I was just being cautious ;)